src/tabs.tsx

```diff
@@ -108,9 +109,18 @@ const Tabs: React.FC<TabsProps> = ({selectedIndex, onChange, tabs, dataAttribute
                                         aria-selected={isSelected ? 'true' : 'false'}
                                     >
                                         {icon && <div className={styles.icon}>{icon}</div>}
-                                        <Text3 medium color="inherit" wordBreak={false} textAlign="center">
+                                        <Text
+                                            desktopSize={textPresets.tabsLabel.size.desktop}
+                                            mobileSize={textPresets.tabsLabel.size.mobile}
+                                            desktopLineHeight={textPresets.tabsLabel.lineHeight.desktop}
+                                            mobileLineHeight={textPresets.tabsLabel.lineHeight.mobile}
+                                            weight="medium"
```

Comment:
If this kind of Texts become frequent, we could update the Text component to directly accept a textPreset